pkgs/development/interpreters/clojure/obb.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl, writeScript }:
+
+stdenv.mkDerivation rec {
+  pname = "obb";
+  version = "0.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/babashka/${pname}/releases/download/v${version}/${pname}.tar.gz";
+    sha256 = "sha256-IgbimykETgUu2ON4OTK6g2oSLvWRnnGV50pJVoiqVk8=";
+  };
```

Comment:
[fetchFromGitHub](https://nixos.org/manual/nixpkgs/stable/#fetchfromgithub) is preferred when we're only fetching the source code

Reply:
The github release is actually a binary. I suppose I actually should be modifying the derivation to compile from source instead and improve reproducibility.